src/components/Photos.js

```diff
@@ -0,0 +1,64 @@
+const Photos = [
+  {
+    id: 1,
+    name: "aries",
+    icon: "./assets/aries.png",
+  },
+  {
+    id: 2,
+    name: "taurus",
+    icon: "./assets/taurus.png",
+  },
+  {
+    id: 3,
+    name: "gemini",
+    icon: "./assets/gemini.png",
+  },
+  {
+    id: 4,
+    name: "cancer",
+    icon: "./assets/cancer.png",
+  },
+  {
+    id: 5,
+    name: "leo",
+    icon: "./assets/leo.png",
+  },
+  {
+    id: 6,
+    name: "virgo",
+    icon: "./assets/virgo.png",
+  },
+  {
+    id: 7,
+    name: "libra",
+    icon: "./assets/libra.png",
+  },
+  {
+    id: 8,
+    name: "scorpio",
+    icon: "./assets/scorpio.png",
+  },
+  {
+    id: 9,
+    name: "sagittarius",
+    icon: "./assets/sagittarius.png",
+  },
+  {
+    id: 10,
+    name: "capricorn",
+    icon: "./assets/capricorn.png",
+  },
+  {
+    id: 11,
+    name: "aquarius",
+    icon: "./assets/aquarius.png",
+  },
+  {
+    id: 12,
+    name: "pisces",
+    icon: "./assets/pisces.png",
+  },
+];
+
```

Comment:
All in all, in my opinion, this is a job well done! I had a lot of "I didn't know you could do that" while looking at your code. And I tried to find something I could suggest for improvement but I could not find one so great job Pauline! I learned some new things as well so thank you!